packages/components/src/Enumeration/Header/HeaderSelected.component.js

```diff
@@ -18,7 +18,7 @@ function getAction(action, index) {
 		}
 	}
 
-	return (
+	return !action.disabled ? (
```

Comment:
you can add a test on disabled and if it is the case return null.
add if(action.disabled) { return null }

Reply:
test already done